mod/proxy.php

```diff
@@ -140,11 +140,23 @@ function proxy_init(App $a) {
 	if (!$direct_cache AND ($cachefile == '')) {
 		$r = qu("SELECT * FROM `photo` WHERE `resource-id` = '%s' LIMIT 1", $urlhash);
 		if (dbm::is_result($r)) {
+<<<<<<< HEAD
```

Comment:
It still is in the change log, which means it'll be merged if the PR is accepted. 🤔